orchagent/mirrororch.cpp

```diff
@@ -583,13 +584,31 @@ void MirrorOrch::setSessionState(const string& name, const MirrorEntry& session,
     if (attr.empty() || attr == MIRROR_SESSION_MONITOR_PORT)
     {
         Port port;
-        m_portsOrch->getPort(session.neighborInfo.portId, port);
+        if ((gMySwitchType == "voq") && (session.type == MIRROR_SESSION_ERSPAN) &&
+            (gIntfsOrch->isRemoteSystemPortIntf(session.neighborInfo.neighbor.alias)))
```

Comment:
Do we need to check if this is RemoteSystemPort here, irrespective of local/remote port we need to use Recirc port ?

Reply:
I think we can remove gIntfsOrch->isRemoteSystemPortIntf(session.neighborInfo.neighbor.alias) check

Reply:
fixed it